components/StyledLinks/index.js

```diff
@@ -2,13 +2,28 @@ import Link from "next/link";
 import styled from "styled-components";
 
 export const StyledLink = styled(Link)`
-  color: black;
+  color: var(--text-color);
   text-decoration: none;
+  font-family: var(--styled-link);
 `;
 
 export const StyledBackLink = styled(StyledLink)`
   display: flex;
-  align-items: center;
   width: 12.5rem;
-  margin: 0.75rem 0 0.75rem -0.3125rem;
+  margin-top: 0.75rem;
+  position: relative;
+`;
+
+export const TransparentBackLink = styled.a`
```

Comment:
A transparent Back Link??? Okay, but please style a Link instead of an `a`- tag.